llama_stack/distribution/start_venv.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
as a follow up I think we could combine the common parts of all the `start_*` scripts... maybe!

Reply:
Merging this for now. This can be a good follow-up PR @cdoern might be interested

Reply:
I am going to open a PR for this today!

Reply:
#1139